lib/polleverywhere/models.rb

```diff
@@ -157,11 +157,11 @@ def self.get(permalink)
       def self.all
         PollEverywhere.http.get.from("/my/polls").as(:json).response do |response|
           ::JSON.parse(response.body).map do |hash|
-            case hash.keys.first.to_sym
+            case hash['type'].to_sym
             when MCP.root_key
               MCP.from_hash(hash)
             when FTP.root_key
-              MCP.from_hash(hash)
+              FTP.from_hash(hash)
```

Comment:
@jsommers I'm happy to merge these changes into the main repo. However, I think the other changes are a bit too specific to how you are using the gem for them to go into the publish rubygem.

Do you think you could change this PR to just include this one commit?

Thank you very much for your contribution.

Reply:
That's fine.  I'll keep my other modifications separate.

Reply:
Either a new PR or changing this one is fine. Thanks!

Reply:
Just created a new one, closed this one.